packages/grafana-data/src/types/logs.ts

```diff
@@ -85,7 +85,8 @@ export interface LogsModel {
   meta?: LogsMetaItem[];
   rows: LogRowModel[];
   series?: DataFrame[];
-  visibleRange?: AbsoluteTimeRange;
+  histogramRange?: AbsoluteTimeRange;
```

Comment:
do you think we should push more the "this is the old approach" here? maybe name it `legacyHistogramRange`? or jus tname it `histogramRange` and deprecate it immediately?

Reply:
well, we didn't say that we are deprecating histogram yet. So if we want to do that, we should do it in 1 PR. But we didn't discuss doing so. 

Keep in mind that histogram is used by all other data sources that don't implement LogVolumes.

Reply:
yeah, i see your point... btw. is this something we need to put into the breaking-changes section?

Reply:
Hmmm, good point. Maybe we shouldn't rename it and just add comment.

Reply:
As we talked about, I have changed the approach and implemented navigationRange calculation in LogsComponent